app/scripts/directives/createSecret.js

```diff
@@ -98,6 +107,9 @@ angular.module("openshiftConsole")
             data: {},
             linkSecret: false,
             pickedServiceAccountToLink: "",
+            keyValuePairs: {
+              data: {}
```

Comment:
I don't see where this data is used when actually creating the secret. Still working on that I'm guessing?

Reply:
Yes, not sure how to include it in the `DataServices.create(secret...`

Reply:
OK, see my comment just below (https://github.com/openshift/origin-web-console/pull/2879#issuecomment-371240087)